generator/metricbeat/{beat}/Makefile

```diff
@@ -24,6 +24,7 @@ setup: copy-vendor git-init
 copy-vendor:
 	mkdir -p vendor/github.com/elastic
 	cp -R ${GOPATH}/src/github.com/elastic/beats vendor/github.com/elastic/
```

Comment:
Oops, I should have applied #13193 also here, could you change it now? :innocent:

Reply:
Ah! Sure.